services/movie_session.py

```diff
@@ -1,44 +1,32 @@
-from django.db.models import QuerySet
-
-from db.models import MovieSession
-
-
-def create_movie_session(
-    movie_show_time: str, movie_id: int, cinema_hall_id: int
-) -> MovieSession:
-    return MovieSession.objects.create(
-        show_time=movie_show_time,
-        movie_id=movie_id,
-        cinema_hall_id=cinema_hall_id,
-    )
-
-
-def get_movies_sessions(session_date: str = None) -> QuerySet:
-    queryset = MovieSession.objects.all()
-    if session_date:
-        queryset = queryset.filter(show_time__date=session_date)
-    return queryset
+from django.db import transaction
+from db.models import Ticket, MovieSession, Movie, CinemaHall
```

Comment:
According to the checklist, it's better to use `get_user_model()` instead of direct model import. Even though `User` model is not used in this file, it's a good practice to follow. Use `from django.contrib.auth import get_user_model` and call `get_user_model()` whenever you need to refer to the User model.